howso/generate_features.amlg

```diff
@@ -698,6 +700,15 @@
 		(map
 			(lambda (append
 				(assoc
+					"shared_deviations"
+						;only need the lags in one feature's attributes. Putting it in the first lag feature to reduce prep processing.
+						(if use_shared_deviations
```

Comment:
since null is the default 'else' clause, you can probably simplify this down to 
(if (and use_shared_deviations (= 1 (current_value 1))) 
    (append ...)
)